src/somd2/config/_config.py

```diff
@@ -185,6 +186,10 @@ def __init__(
             Power to use for the soft-core Coulomb interaction. This is used
             to soften the electrostatic interaction.
 
+        shift_couloomb : str
```

Comment:
`couloomb` should be `coulomb`

Reply:
Nice to know that co-pilot make typos too :-)